pom.xml

```diff
@@ -32,8 +32,8 @@
         <!-- Versioning -->
         <version.arquillian_core>1.1.11.Final</version.arquillian_core>
         
-        <weld.api.version>2.3.SP2</weld.api.version>
-        <weld.core.version>2.3.5.Final</weld.core.version>
+        <weld.api.version>2.4.CR1</weld.api.version>
+        <weld.core.version>2.4.0.CR1</weld.core.version>
```

Comment:
I'm not sure I get the concern to its full extent, but if I understand @johnament it means that he, while being on Weld 2.3.x.Final, wouldn;t be able to use Weld Container adapter 2.0.x as it's build against newer version of Weld. Maybe we should make versioning more aligned with Weld versions then? Would that solve anything?